ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +342,55 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_node_utilization(self, node_type=('worker', 'master')):
```

Comment:
This function can be used in other tests also. Please move it out to a suitable common location

Reply:
ocs_ci/ocs/node.py 
Is ok?

Reply:
@hnallurv @ebenahar @PrasadDesala @vasukulkarni

Reply:
Added my comment in the new commit.